src/CONST.ts

```diff
@@ -4779,7 +4779,7 @@ const CONST = {
     SEARCH_RESULTS_PAGE_SIZE: 50,
 
     SEARCH_DATA_TYPES: {
-        TRANSACTION: 'transaction',
+        TRANSACTION: 'transactions',
```

Comment:
Will this be proper name or will we revert back to singular? @luacmartins

Reply:
Yes, the correct value is `transaction`